js/main.js

```diff
@@ -128,7 +128,10 @@ document.documentElement.classList.add('hide-app')
 window.addEventListener(
   'load',
   e => {
-    l10nReady.then(main)
+    l10nReady.then(main);
+    
+    if (document.getElementById("version-info").innerHTML == ("{" + "VERSION" + "}"))
```

Comment:
`innerHTML` isn't necessary; should use `textContent`

`==` is eeeeeeee

parentheses not needed around concatenation

this check could be done in the script tag in appdesign.html which gets removed when it's built